llama_index/schema.py

```diff
@@ -112,10 +215,110 @@ def get_embedding(self) -> List[float]:
             raise ValueError("embedding not set.")
         return self.embedding
 
+
+class TextNode(BaseNode):
```

Comment:
curious: how do we reload a text node after retrieving e.g. from vector db?

Reply:
still need to implement that lol buuuut since it's all pydantic now, should hopefully be a little more straightforward

Reply:
I see, you are thinking about serializing the whole thing into a json string?

Reply:
Pretty much, either that or using dict like we kind of already do